src/Chapter2.hs

```diff
@@ -738,9 +750,10 @@ the list with only those lists that contain a passed element.
 >>> contains 3 [[1, 2, 3, 4, 5], [2, 0], [3, 4]]
 [[1,2,3,4,5],[3,4]]
 
-🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
+🕯 HINT: Use the 'elem' function to check whether an elq2 ement belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains el l = filter (\sub_l -> el `elem` sub_l) l
```

Comment:
Now, after you've mastered eta-reduction, you can apply such a technique to other functions as well  🙂 Here you have two opportunities for eta-reduction: lambda and the last argument for `filter`
But your implementation is already great 💯